### ISSUE TITLE
Simulate hypothetical pools

### ISSUE THREAD

**Issue by UmerHA** [curveresearch/curvesim#74]
I'm creating functionality to enable loading historic pool volumes from local files This would enable simulating hypothetical pools.

From your perspective, is there a reason against that?

**Comment by nagakingg** [comment]
Good point. Currently, the historical volume is just summed up across the last 2 months, so maybe the right thing to do is to have an optional argument where one could input a single float for historical volume?

Currently, there is an optional vol_mult argument, which is just a ratio (or vector of ratios for each token-pair) of pool_volume/total_market_volume. But, there's no reason a user would know the total market volume, so I think maybe either argument (vol_mult or something like historical_volume) but not both should be available.

Thoughts @chanhosuh ?

**Comment by chanhosuh** [comment]
I think just passing an optional arg, like naga suggested, is the simplest and easiest for a typical user.

**Comment by chanhosuh** [comment]
The way the volume data travels through the framework could be improved though.  We're looking to remove a lot of functionality from the `PoolData` thing so this might be another thing we'll have to rethink.
